temba/contacts/models.py

```diff
@@ -1893,7 +1893,7 @@ def send(self, text, user, trigger_send=True, response_to=None, expressions_cont
                 msg = Msg.create_outgoing(self.org, user, recipient, text,
                                           response_to=response_to, expressions_context=expressions_context, connection=connection,
                                           attachments=attachments, msg_type=msg_type or INBOX, status=status,
-                                          created_on=created_on, high_priority=high_priority)
+                                          metadata=quick_replies, created_on=created_on, high_priority=high_priority)
```

Comment:
create_outgoing should have `quick_replies` as a parameter. We shouldn't ever see `metadata=quick_replies`